### PR TITLE
Packaging

### PR DESCRIPTION
**Pull request by rpiaggio**

### CODE REVIEW

File: modules/web/client/src/main/scala/observe/ui/components/MainApp.scala [hunk 1]
```diff
@@ -136,14 +139,15 @@ object MainApp extends ServerEventHandler:
...
-        // .map(_.orElse(throw new Exception("User is not staff")))
-        case _                                     => IO(userVault)
+    ssoClient.whoami
+      .flatMap: userVault =>
+        userVault.map(_.user) match
+          case Some(StandardUser(_, role, other, _)) =>
+            (role +: other)
+              .collectFirst { case StandardRole.Staff(roleId) => roleId }
+              .fold(IO(userVault))(ssoClient.switchRole)
+          // .map(_.orElse(throw new Exception("User is not staff")))
```
Comment (cquiroz) [inline modules/web/client/src/main/scala/observe/ui/components/MainApp.scala:149]: Commented code

File: modules/web/client/src/main/scala/observe/ui/components/services/ObservationSyncer.scala [hunk 2]
```diff
@@ -71,30 +67,32 @@ object ObservationSyncer:
...
 
-                  val initialBreakpoints: Set[Step.Id] =
-                    config match
-                      case InstrumentExecutionConfig.GmosNorth(executionConfig) =>
-                        getBreakPoints(executionConfig.acquisition) ++
-                          getBreakPoints(executionConfig.science)
-                      case InstrumentExecutionConfig.GmosSouth(executionConfig) =>
-                        getBreakPoints(executionConfig.acquisition) ++
-                          getBreakPoints(executionConfig.science)
+            //     val initialBreakpoints: Set[Step.Id] =
```
Comment (cquiroz) [inline modules/web/client/src/main/scala/observe/ui/components/services/ObservationSyncer.scala:84]: Is this something that will be fixed later on?

File: modules/web/server/src/main/resources/app.conf [hunk 3]
```diff
@@ -44,6 +44,12 @@ web-server {
     insecure-port = 7071
     # External url used for redirects
     external-base-url = "localhost"
+
+    tls {
+        key-store = "src/main/resources/cacerts.jks"
```
Comment (cquiroz) [inline modules/web/server/src/main/resources/app.conf:49]: are you including a self-signed cert?

File: modules/web/server/src/main/scala/observe/web/server/http4s/StaticRoutes.scala [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+  def localFile(path: String, req: Request[F]): OptionT[F, Response[F]] =
+    // Get full path to JAR/class
+    // https://stackoverflow.com/questions/320542/how-to-get-the-path-of-a-running-jar-file
+    val runningPath = Paths.get(getClass.getProtectionDomain.getCodeSource.getLocation.toURI)
+    // Get the current directory (should be "lib") and then go to the "app" sibling
+    val fullPath    = runningPath.getParent.resolveSibling(AppDir).resolve(path.stripPrefix("/"))
+    StaticFile.fromPath(Path.fromNioPath(fullPath), req.some) // .map(_.putHeaders())
+
+  implicit class ReqOps(req: Request[F]) {
```
Comment (cquiroz) [inline modules/web/server/src/main/scala/observe/web/server/http4s/StaticRoutes.scala:43]: No extension?

**Review by cquiroz — APPROVED** [review]
LGTM

File: modules/web/client/src/main/scala/observe/ui/components/MainApp.scala (hunk 1 above)
Comment (rpiaggio) [reply modules/web/client/src/main/scala/observe/ui/components/MainApp.scala:149]: Thank you. This one is on purpose. We may have to find a better mechanism to handle this. Will the STAFF role be enough to operate oberve?

File: modules/web/client/src/main/scala/observe/ui/components/services/ObservationSyncer.scala (hunk 2 above)
Comment (rpiaggio) [reply modules/web/client/src/main/scala/observe/ui/components/services/ObservationSyncer.scala:84]: We should move that code to the server.

File: modules/web/server/src/main/resources/app.conf (hunk 3 above)
Comment (rpiaggio) [reply modules/web/server/src/main/resources/app.conf]: Yup, I now named it `cacerts.jks.dev`.

File: modules/web/server/src/main/scala/observe/web/server/http4s/StaticRoutes.scala (hunk 4 above)
Comment (rpiaggio) [reply modules/web/server/src/main/scala/observe/web/server/http4s/StaticRoutes.scala:43]: WDYM?

Comment (cquiroz) [reply modules/web/server/src/main/scala/observe/web/server/http4s/StaticRoutes.scala:43]: Could this be an extension method?

Comment (rpiaggio) [reply modules/web/server/src/main/scala/observe/web/server/http4s/StaticRoutes.scala:43]: Ah, yes! This was copied from seqexec. I'll migrate.

**Pull request merged**